jupyter-config/server-config/spark_connect_labextension.json

```diff
@@ -0,0 +1,69 @@
+{
+  "ServerApp": {
+    "jpserver_extensions": {
+      "spark_connect_labextension": true
+    }
+  },
+  "SparkConnectConfig": {
```

Comment:
This file is for development purposes only, and may also be for reference. I don't think this file will be automatically included when installing.

Reply:
Since we would need the admins to configure the extension with their own bundles and options, I suppose it's better to remove this completely, wdyt?  @diocas @etejedor

Reply:
I believe there are two things here, one is the automatic enabling of the extension, which iiuc @diocas wants to keep. This can stay alone in this file.

Then we have the part of the clusters, the bundles and the Spark options. This will come from some configuration files on CVMFS, and we should have some configuration options to point to it, i.e. three options, one for the path containing the definition of the clusters, one for the bundles and one for the Spark options). In the extension itself (or in this PR), we can leave example files (e.g. bundles.json, clusters.json, spark_options.json) of how they would look like.

Now I see also the error_suggestions, which could be yet another configurable option and example file.

Reply:
So just keep the file like this?
```json
{
  "ServerApp": {
    "jpserver_extensions": {
      "spark_connect_labextension": true
    }
  }
}
```

Reply:
Exactly this!